src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -500,7 +500,10 @@ def register_submission_to_objects_api(
             submission, object_mapping, REGISTRATION_ATTRIBUTE, object_data
         )
 
-        with get_objects_client() as objects_client:
+        # In a follow up PR: the group will be configurable:
+        with get_objects_client(
+            ObjectsAPIGroupConfig.objects.first()
```

Comment:
```suggestion
            ObjectsAPIGroupConfig.objects.order_by("pk").first()
```
let's make this explicit